.github/SUPPORT.md

```diff
@@ -0,0 +1,11 @@
+OpenSSL User Support resources
+==============================
+
+For *questions* on how to use OpenSSL or what went wrong when you
+tried something, our primary resource is the mailing list
+openssl-users@openssl.org, where you can get help from others in the
+OpenSSL community (which includes the developers as time permits).
+
+openssl-users@openssl.org is a subscribers only list, so you will have
```

Comment:
Only subscribers can post to this list (although the archives are public).  See ...